src/components/clubdetail/InviteMemberCard/InviteMemberCard.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState, ChangeEvent } from 'react';
+import clsx from 'clsx';
+
+import { inviteMemberByEmail } from '../../../api/clubsMembers';
+import { ClubDetailCardWrapper } from '../ClubDetailCardWrapper/ClubDetailCardWrapper';
+import style from './InviteMemberCard.module.scss';
+
+type InviteMemberCardProps = {
+  title: string;
+  text: string;
+  clubId: number;
+  loadClubMembers: () => Promise<void>;
+};
+
+export const InviteMemberCard: React.FC<InviteMemberCardProps> = ({
+  title,
+  text,
+  clubId,
+  loadClubMembers,
+}) => {
+  const [email, setEmail] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [successMessage, setSuccessMessage] = useState('');
+
+  const validateEmail = (emailValidation: string) =>
+    /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(emailValidation);
+
+  const handleEmailChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setEmail(e.target.value);
+    setErrorMessage('');
+  };
+
+  const handleInvite = async () => {
+    setErrorMessage('');
+    if (!email.trim()) {
+      setErrorMessage('Please enter an email');
+
+      return;
+    }
+
+    if (!validateEmail(email.trim())) {
```

Comment:
What about making it more local?

```suggestion
  const validateEmail =  /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test
    if (!validateEmail(email.trim())) {
```

Reply:
the .test is intentionally NOT called